bundles/org.openhab.ui/web/src/pages/settings/settings-menu.vue

```diff
@@ -201,7 +201,7 @@ export default {
       // can be done in parallel!
       servicesPromise.then((data) => {
         this.systemServices = data.filter(s => s.category === 'system')
-        this.otherServices = data.filter(s => s.category !== 'system')
+        this.otherServices = data.filter(s => s.category !== 'system' && s.category !== 'persistence')
```

Comment:
FYI I tested this to filter out all add-ons:

```suggestion
        this.otherServices = data.filter(s => !['system', 'automation', 'bindings', 'io', 'persistence', 'transformations', 'ui', 'voice'].includes(s.category))
```

Reply:
> Problematic is the rule-based speech intepreter (`org.openhab.rulehli`) because is is provided by core and therefore no add-on, but has the category `voice`. Even though it belongs to voice, it should be changed to `misc` or `system` because it is provided by openHAB core.

I agree. I also think that LSP should be system, because that is probably the only non-add-on service left.

Reply:
Maybe the rule-based speech intepreter could become an add-on?

Reply:
LSP and rule-based speech intepreter are the only non-add-on services left (AFAIK), for LSP I agree that it should be `system` and for rule-bases sp...ter I would say either `system` or `misc`. These two options would make keep them visible in MainUI. 

Rule-based sp..ter would be in the system settings section with `system` or the only one in the Other Services section with `misc`.

Reply:
The most important is to not loose settings in existing user setups. 
Then showing them in system or misc is a detail.